source/main/gui/panels/GUI_MainSelector.cpp

```diff
@@ -880,14 +881,25 @@ void CLASS::SetPreviewImage(Ogre::String texture)
 	}
 	catch (...)
 	{
+		Ogre::LogManager::getSingleton().stream() << "[RoR|SelectorGUI] Failed to load preview image: " << texture;
 		m_Preview->setVisible(false);
 	}
 }
 
 void CLASS::ResizePreviewImage()
 {
 	MyGUI::IntSize imgSize(0, 0);
-	Ogre::TexturePtr t = Ogre::TextureManager::getSingleton().load(m_preview_image_texture, Ogre::ResourceGroupManager::AUTODETECT_RESOURCE_GROUP_NAME);
+	Ogre::TexturePtr t;
+	try
+	{
+		t = Ogre::TextureManager::getSingleton().load(m_preview_image_texture, Ogre::ResourceGroupManager::AUTODETECT_RESOURCE_GROUP_NAME);
+	}
+	catch (...)
```

Comment:
This won't work. This code: https://github.com/only-a-ptr/rigs-of-rods/blob/99d734803b5448f492b164ce0ca6ad0308a076b8/source/main/gui/panels/GUI_MainSelector.cpp#L879-L880
would be executed after catching everything here.